homeassistant/components/binary_sensor/__init__.py

```diff
@@ -9,7 +9,7 @@
 
 from homeassistant.helpers.entity_component import EntityComponent
 from homeassistant.helpers.entity import Entity
-from homeassistant.const import (STATE_ON, STATE_OFF)
+from homeassistant.const import (ATTR_SENSOR_CLASS, STATE_ON, STATE_OFF)
```

Comment:
I think it's fine to keep binary sensor class in the binary component. It is not going to be used by other components